Client/Frontend/Browser/Tab.swift

```diff
@@ -548,8 +559,15 @@ class Tab: NSObject {
             return
         }
 
+        updatePullToRefreshVisibility()
```

Comment:
Aren't we already covering this case inside BVC `preferencesDidChange` ?

Reply:
No this call happens whenever a URL changes in the tab's web view (since we dont want pull to refresh on local pages at all). The BVC pref observation only ensures that it updates any visible tabs refresh control when a user flips that pref on/off